<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
      <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
          <widget class="QLabel" name="labelCardsPerDay">
            <property name="text"><string>Cards per day:</string></property>
          </widget>
        </item>
        <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxCardsPerDay">
            <property name="minimum"><number>1</number></property>
            <property name="maximum"><number>100</number></property>
          </widget>
        </item>
        <item row="1" column="0">
          <widget class="QLabel" name="labelLearnThreshold">
            <property name="text"><string>Learn threshold:</string></property>
          </widget>
        </item>
        <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxLearnThreshold">
            <property name="minimum"><number>1</number></property>
            <property name="maximum"><number>10</number></property>
          </widget>
        </item>
      </layout>
    </item>
    <item>
      <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
        <property name="standardButtons"><set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set></property>
      </widget>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
